src/tracing.h

```diff
@@ -0,0 +1,82 @@
+/****************************************************************************
+**
+**  This file is part of GAP, a system for computational discrete algebra.
+**
+**  Copyright of GAP belongs to its developers, whose names are too numerous
+**  to list here. Please refer to the COPYRIGHT file for details.
+**
+**  SPDX-License-Identifier: GPL-2.0-or-later
+**
+**  This file defines the functions for tracing operations.
+*/
+
+#include "objects.h"
+
+
+void ReportWrappedOperation1(const char *, Obj op);
+void ReportWrappedOperation2(const char *, Obj op1, Obj op2);
+void InstallOpWrapper(voidfunc, voidfunc);
+
+
+// These are macros which make it simple to wrap
```

Comment:
This comment looks as if it is incomplete? In any case, it is useless in the current form. Would be nice to have at least some comments explaining what those macros are and how to use them.